### ISSUE TITLE
Yandex not working 

### ISSUE THREAD

**Issue by arsium** [0Fdemir/Chromium-Password-Recovery#1]
Hello sir I've tested your project (and I added new browsers working I've tested) but yandex isn't working. Find this on xenarmor : https://xenarmor.com/how-to-recover-saved-passwords-in-yandex-browser/

**Comment by 0xfd3** [comment]
Yes i know. Yandex updated its storage algorithm a long time ago. I will try to look at it at a convenient time.
